server/views/pages/adjustments/hub.njk

```diff
@@ -90,7 +90,7 @@
                 Update these details
               </li>
               <li>
-                Reload this page
+                <a href=".">Reload this page</a>
```

Comment:
I think everywhere else we use an empty string to reload the page.

Reply:
Adjusted to use empty string